src/components/SearchResults.js

```diff
@@ -11,14 +11,22 @@ function SearchResults({ results }) {
             <th scope="col">FirstName</th>
             <th scope="col">Surname</th>
             <th scope="col">Room Id</th>
-            <th scope="col">`check in date</th>
-            <th scope="col">check out date</th>
+            <th scope="col">Check In Date</th>
+            <th scope="col">Check Out Date</th>
           </tr>
         </thead>
         <tbody>
           {results.map((person, index) => {
+            const [select, setSelect] = useState(true);
+            function handleClick() {
+              setSelect(select => !select);
+            }
             return (
-              <tr key={index}>
+              <tr
+                key={index}
+                onClick={handleClick}
+                style={{ backgroundColor: select ? "" : "green" }}
```

Comment:
You can use `style` directly for something simple like this.

As the site gets larger and more complex, you'll probably have other tables which need rows to be highlighted. In that case you'll want to define a `selectedRow` css class once and then reuse it everywhere you need to highlight table rows (using `className` in React or just `class` if you're writing plain html). Then you can change the style of all tables at once, instead of the `style` property of every table in the site.

If you run the app, the grey text is quite hard to read against the dark green background, which isn't the best for usability. Maybe change the text colour to white, or make the green colour much lighter so the text still pops?

Reply:
Ah, in fact bootstrap already has some classes you can use for this.

https://getbootstrap.com/docs/4.0/content/tables/#contextual-classes